src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Mocks/IVsFileChangeExFactory.cs

```diff
@@ -1,8 +1,9 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using Microsoft.VisualStudio.Shell.Interop;
 using Moq;
 
-namespace Microsoft.VisualStudio.Shell.Interop
+namespace Microsoft.VisualStudio.ProjectSystem
```

Comment:
Why? I know Bill wrote this one, but I do this very deliberately - I put them in the namespace, alongside the thing they are creating.